openedx/adg/common/tests/test_helpers.py

```diff
@@ -0,0 +1,18 @@
+"""
+Common helper tests
+"""
+import pytest
+
+from openedx.adg.common.helpers import convert_float_point_to_percentage
+
+
+@pytest.mark.parametrize('grade_percent, expected_grade_percent', [
+    (1.0, 100),
+    (0.0, 0),
+])
+def test_convert_float_point_to_percentage(grade_percent, expected_grade_percent):
+    """
+    Test `convert_float_point_to_percentage` return expected grade percentage value.
+    """
+    actual_percetage = convert_float_point_to_percentage(grade_percent)
```

Comment:
percentage*

Reply:
Test that `convert_float_point_to_percentage` returns expected grade percentage value.